steps/generate_dataset_from_htme.py

```diff
@@ -155,15 +174,20 @@ def consolidate_rdd_per_collection(collection, secrets_collections, s3_client,
                 json_location_prefix
             )
             persist_json(json_location, consolidated_rdd_mapped)
-            the_logger.info("Applying Tags for prefix : %s for correlation id : %s ", json_location_prefix, args.correlation_id)
+            the_logger.info("Applying Tags for prefix : %s for correlation id : %s ", json_location_prefix,
+                            args.correlation_id)
             tag_objects(json_location_prefix, tag_value, s3_client, s3_publish_bucket)
-        the_logger.info("Creating Hive tables of collection : %s for correlation id : %s", collection_name, args.correlation_id)
+        the_logger.info("Creating Hive tables of collection : %s for correlation id : %s", collection_name,
+                        args.correlation_id)
         end_time = time.perf_counter()
         total_time = round(end_time - start_time)
         add_metric("processing_times.csv", collection_name, str(total_time))
-        the_logger.info("Completed Processing of collection : %s for correlation id : %s", collection_name, args.correlation_id)
+        the_logger.info("Completed Processing of collection : %s for correlation id : %s", collection_name,
+                        args.correlation_id)
     except BaseException as ex:
-        the_logger.error(f"Error processing collection for correlation id: %s for collection %s %s", args.correlation_id, collection_name, str(ex))
+        the_logger.error(f"Error processing collection for correlation id: %s for collection %s %s",
```

Comment:
F string can be removed here as the logger can replace the variables

Reply:
I think this was your change I haven't changed this line of code I have only reformatted